scripts/publish-impl

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# publishSigned or publishLocal
+publish=$1
+
+set -ex
+
+sbt clean
+
+# lowest versions supported
+for v in 2.11.12 2.12.13 2.13.4; do
```

Comment:
We should use the latest version instead, in this case.

Reply:
Thank-you, I will fix that. Do you know why `publishSigned` is basically not found? I haven't looked to see if there is a change with something in sbt.

Reply:
Because sbt-pgp is not in `project/build.sbt`. It's in the global `plugins.sbt` of the machine running the publishing.

Reply:
I updated the docs so it should be good to go once CI completes.